From 6882aba429b4a49cb4312047f617dce1ed976147 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 24 Nov 2023 10:52:26 +0100
Subject: [PATCH] Apply TBB LTO patch

Originally created by Elia Devito <eliadevito@gmail.com> for
PrusaSlicer.
---
 deps/TBB/TBB.cmake | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/deps/TBB/TBB.cmake b/deps/TBB/TBB.cmake
index 87b76828a8aa..c680690d41f3 100644
--- a/deps/TBB/TBB.cmake
+++ b/deps/TBB/TBB.cmake
@@ -1,8 +1,10 @@
+set(_patch_command ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_LIST_DIR}/GNU.cmake cmake/compilers/GNU.cmake)
+
 bambustudio_add_cmake_project(
     TBB
     URL "https://github.com/oneapi-src/oneTBB/archive/refs/tags/v2021.5.0.zip"
     URL_HASH SHA256=83ea786c964a384dd72534f9854b419716f412f9d43c0be88d41874763e7bb47
-    #PATCH_COMMAND ${PATCH_CMD} ${CMAKE_CURRENT_LIST_DIR}/0001-TBB-GCC13.patch
+    PATCH_COMMAND ${_patch_command}
     CMAKE_ARGS          
         -DTBB_BUILD_SHARED=OFF
         -DTBB_BUILD_TESTS=OFF
-- 
2.43.0

